internal/plugin/plugin.go

```diff
@@ -25,17 +26,19 @@ import (
 // TODO add cancel and async (done context) support
 
 const LATEST = "latest"
+
 var nameRegexp = regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9]$`)
 var versionRegexp = regexp.MustCompile(`^[0-9]+\.[0-9]+\.[0-9]+$`)
 
-var localPluginFolder = "./raito/plugins/"
+var localPluginFolder = "~/.raito/plugins/"
```

Comment:
> Why did you change this? The local plugin folder should be local to the current directory. The global one is in the user home.

That's where the plugins ended up for me, but I can adapt that locally I guess. I was trying to avoid committing this change, but I did a bad job apparently. I'll revert.